tests/conftest.py

```diff
@@ -190,3 +190,11 @@ def git_erepo():
     repo.setUp()
     yield repo
     repo.tearDown()
+
+
+@pytest.fixture
+def empty_dir():
```

Comment:
see this comment: https://github.com/iterative/dvc/pull/2852/files#r352164905

Reply:
This won't be needed when new dir_helper fixtures land. So maybe save effort on this refactoring. If you simply need an empty dir you may use `tmp_path` pytest standard fixture and `monkeypatch` to chdir to it:
```python
@pytest.fixture
def empty_dir(tmp_path, monkeypatch):
    monkeypatch.chdir(tmp_path)
    return tmp_path

```